From 3f166417b837d6fb44e1003a2286876cb5c682b4 Mon Sep 17 00:00:00 2001
From: "Patil, Rachna" <rachna@ti.com>
Date: Thu, 26 Apr 2012 15:20:20 +0530
Subject: [PATCH 67/68] video: da8xx-fb: Add Newhaven LCD Panel details

This patch adds details for Newhaven display Panel.
This panel is used on AM335x EVM-SK An upcoming TI EVM.

Signed-off-by: Patil, Rachna <rachna@ti.com>

Conflicts:

	drivers/video/da8xx-fb.c
---
 drivers/video/da8xx-fb.c |   14 ++++++++++++++
 1 files changed, 14 insertions(+), 0 deletions(-)

diff --git a/drivers/video/da8xx-fb.c b/drivers/video/da8xx-fb.c
index 9aaca5d..010a8bc 100644
--- a/drivers/video/da8xx-fb.c
+++ b/drivers/video/da8xx-fb.c
@@ -301,6 +301,20 @@ static struct da8xx_panel known_lcd_panels[] = {
 		.pxl_clk = 8000000,
 		.invert_pxl_clk = 0,
 	},	
+	/* Newhaven Display */
+	[5] = {
+		.name = "NHD-4.3-ATXI#-T-1",
+		.width = 480,
+		.height = 272,
+		.hfp = 8,
+		.hbp = 43,
+		.hsw = 4,
+		.vfp = 4,
+		.vbp = 12,
+		.vsw = 10,
+		.pxl_clk = 9000000,
+		.invert_pxl_clk = 0,
+	},
 };
 
 /* Enable the Raster Engine of the LCD Controller */
-- 
1.7.7.6

